src/goDebug.ts

```diff
@@ -0,0 +1,67 @@
+import * as semver from 'semver';
+import * as vscode from 'vscode';
+
+import * as plz from './please';
+import { getBinPath } from './utils/pathUtils';
+
+const PLZ_MIN_VERSION = '16.1.0-beta.4';
+
+const outputChannel = vscode.window.createOutputChannel('Please Go Tests');
+
+export async function debug(document: vscode.TextDocument, functionName = '') {
+    const { fileName } = document;
+    if (!fileName.endsWith('_test.go')) {
+        throw new Error(`Not a test file: ${fileName}`);
+    }
+
+    await document.save();
+
+    outputChannel.clear();
+    outputChannel.appendLine('Starting up tests:')
+    outputChannel.show();
+
+    const debugConfig: vscode.DebugConfiguration = {
+        type: 'plz-go',
+        request: 'launch',
+        name: 'Launch test',
+    };
+
+    outputChannel.appendLine('Getting Please version...');
+    if (semver.lt(await plz.version(), PLZ_MIN_VERSION)) {
+        throw new Error(`You need to be at least on Please version ${PLZ_MIN_VERSION}. Go to https://github.com/thought-machine/please to update it.`);
+    }
+
+    debugConfig['plzBinPath'] = plz.binPath();
+
+    const dlvBinPath = getBinPath('dlv');
```

Comment:
I guess these need updating too. `getBinPath()` will just return dlv here.

Reply:
I'll just fix that third party utility